crates/tx5/src/ep.rs

```diff
@@ -272,8 +272,9 @@ impl Endpoint {
                 let data = data.to_vec();
                 tokio::task::spawn(async move {
                     let _ = tokio::time::timeout(timeout, async {
-                        peer.ready().await;
-                        let _ = peer.send(data).await;
+                        if let Some(conn) = peer.wait_for_ready().await {
+                            let _ = conn.send(data).await;
+                        }
```

Comment:
We don't log individual errors here on broadcast. Again, debatable but also potentially very noisy on a larger network.